Makefile

```diff
@@ -0,0 +1,20 @@
+platform:
+	PROFILE=platform docker compose --profile platform up --attach conductor
+
+stageDev:
+	PROFILE=stageDev docker compose --profile stageDev up --attach conductor
+
+arrangerDev:
+	PROFILE=arrangerDev docker compose --profile arrangerDev up --attach conductor
+
+maestroDev:
+	PROFILE=maestroDev docker compose --profile maestroDev up --attach conductor
+
+songDev:
+	PROFILE=songDev docker compose --profile songDev up --attach conductor
+
+scoreDev:
+	PROFILE=scoreDev docker compose --profile scoreDev up --attach conductor
+
+down:
```

Comment:
lazy question: does this work for all of the above? hoping the answer is yes, otherwise we'll need to rename it "platform down", and add "up" to all others (possibly with their own downs) or just remove this one to keep things consistent.